File: C++/1893.cpp
#include <iostream>
#include <string>

int main() {
  std::string seat_number;
  std::cin >> seat_number;

  auto letter = seat_number.back();
  auto number = std::stoi(seat_number.substr(0, seat_number.size() - 1));

  std::string result = "aisle";
  if (letter == 'A' || (number <= 2 && letter == 'D') ||
      (3 <= number && number <= 20 && letter == 'F') ||
      (21 <= number && letter == 'K')) {
    result = "window";
  } else if (21 <= number && (letter == 'B' || letter == 'E' || letter == 'F' ||
                              letter == 'J')) {
    result = "neither";
  }
  std::cout << result << '\n';
  return 0;
}

File: Python/1893.py
seat_number = input()

letter = seat_number[-1]
number = int(seat_number[:-1])

result = 'aisle'
if letter == 'A' or (number <= 2 and letter == 'D') \
        or (3 <= number <= 20 and letter == 'F') \
        or (21 <= number and letter == 'K'):
    result = "window"
elif 21 <= number and  letter in 'BEFJ':
    result = "neither"
print(result)
